### PR TITLE
Create pod

### PR DESCRIPTION
**Pull request by chloeprezelski**
Allows user to create a pod if they are not in one already.

### CODE REVIEW

File: mobile/MyApp/App.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
+/**
```
Comment (allysmith12) [inline mobile/MyApp/App.tsx]: Should this file be in the PR?

File: mobile/src/pods/PodsNavigator.tsx [hunk 2]
```diff
@@ -8,79 +8,28 @@ import {
...
-                  setCreatePodClicked(true);
-                  return;
-                }}
-              ></Button>
-            )}
-          </View>
-        )}
-      </View>
-    </SafeAreaView>
+    <Stack.Navigator>
```
Comment (allysmith12) [inline mobile/src/pods/PodsNavigator.tsx:28]: Ooo nice!! We did the same thing with the Stack.Navigator

**Review by allysmith12 — COMMENTED** [review]
There's a bunch of files in here that appear to be apart of MyApp (an example application). I'm not sure how they got in here but should probably be removed! Otherwise the pod stuff looks good :)

File: mobile/MyApp/App.tsx (hunk 1 above)
Comment (chloeprezelski) [reply mobile/MyApp/App.tsx]: I removed it. Thanks for pointing this out :)

**Comment by allysmith12** [comment]
Looks great!!

**Review by allysmith12 — APPROVED** [review]
LGTM!

**Pull request merged**